app/client/cli/helpers/common.go

```diff
@@ -10,5 +19,44 @@ var (
 	genesisPath = runtime.GetEnv("GENESIS_PATH", "build/config/genesis.json")
 
 	// P2PMod is initialized in order to broadcast a message to the local network
+	// TECHDEBT: prefer to retrieve P2P module from the bus instead.
 	P2PMod modules.P2PModule
 )
+
+// fetchPeerstore retrieves the providers from the CLI context and uses them to retrieve the address book for the current height
```

Comment:
Comment is `fetchPeerstore` but function is `FetchPeerstore`. What are we aiming for?